src/components/movies/MovieDetail.vue

```diff
@@ -1,26 +1,136 @@
 <script>
 import { defineComponent } from 'vue';
+import { mapActions } from 'pinia';
+import { mainStore } from '@/store/index.js';
+
+import BaseTag from '@/components/base/BaseTag.vue';
 
 export default defineComponent({
+  components: {
+    BaseTag,
+  },
   props: {
     movie: {
       type: Object,
       required: true,
     },
   },
+  computed: {
+    imageStyles() {
+      let backgroundPosition;
+
+      if (this.movie.id === 1) backgroundPosition = '50% 30%';
+      else if (this.movie.id === 2 || this.movie.id === 8) backgroundPosition = '50% 65%';
+      else if (this.movie.id === 3) backgroundPosition = '50% 85%';
+      else if (this.movie.id === 4) backgroundPosition = '50% 55%';
+      else if (this.movie.id === 5) backgroundPosition = '50% 60%';
+      else if (this.movie.id === 6 || this.movie.id === 9) backgroundPosition = '50% 15%';
+      else if (this.movie.id === 7) backgroundPosition = '50% 93%';
+      else if (this.movie.id === 10) backgroundPosition = '50% 45%';
+      else if (this.movie.id === 11) backgroundPosition = '50% 25%';
+      else backgroundPosition = '50% 50%';
+
+      return {
+        'background-image': `url(${this.movie.poster_url})`,
+        'background-position': backgroundPosition,
+      };
+    },
+    movieReleaseYear() {
+      const indexOfFirstHyphen = this.movie.release_date.indexOf('-');
+      return this.movie.release_date.slice(0, indexOfFirstHyphen);
```

Comment:
I think a bit less hacky solution could be to simply parse your `release_date` to `Date` object and then retrieve a year

```suggestion
      const date = new Date(this.move.release_date);
      return date.getFullYear();
```